src/main/java/chylex/hee/mechanics/voidchest/InventoryVoidChest.java

```diff
@@ -1,78 +1,86 @@
 package chylex.hee.mechanics.voidchest;
-import java.util.Random;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.inventory.InventoryBasic;
 import net.minecraft.item.ItemStack;
 import chylex.hee.tileentity.TileEntityVoidChest;
 
-public class InventoryVoidChest extends InventoryBasic{
+public class InventoryVoidChest extends InventoryBasic {
 	private TileEntityVoidChest chest;
 
-	public InventoryVoidChest(){
-		super("container.voidChest",false,27);
+	public InventoryVoidChest() {
+		super("container.voidChest", false, 27);
 	}
-	
-	public InventoryVoidChest setChest(TileEntityVoidChest chest){
+
+	public InventoryVoidChest setChest(TileEntityVoidChest chest) {
 		this.chest = chest;
 		return this;
 	}
-	
-	public void putItemRandomly(ItemStack is, Random rand){
+
+	private static boolean isSameItem(ItemStack dis, ItemStack other) {
+		if (dis == null && other != null || dis != null && other == null)
+			return false;
+		return dis == other || dis.getItem() == other.getItem() && dis.getItemDamage() == other.getItemDamage() && ItemStack.areItemStackTagsEqual(dis, other);
```

Comment:
isent the chance of dis == other basicly zere sinse they have to be the same object

also why not use are areItemStacksEqual instead of 3 and statments

Reply:
well yes. appox zero, but not 0.

areItemStacksEqual will also compare stack sizes.